packages/synapse-react-client/src/synapse-queries/KeyFactory.ts

```diff
@@ -122,7 +122,7 @@ const entityQueryKeyObjects = {
     infinite: boolean,
   ) => [
     entityQueryKeyObjects.entity(queryBundleRequest.entityId),
-    'tableQueryResultWithAsyncStatus',
+    { type: 'tableQueryResult', withAsyncStatus: true },
```

Comment:
Update the `tableQueryResult` query keys to all match so they can all be invalidated at once, when necessary